### DISCUSSION TITLE
I need 256 colors in IBM OS/2 2.1 in 86box

### DISCUSSION THREAD

**Discussion by MAAR2003** [86Box/86Box#1293] [Q&A] [answered]
Hello! I'm new in here! By the way, is there a download link of any grahpic driver in which I get 256 colors in IBM OS/2.1 that works in 86Box? If you have a link of it, thank you!

**Comment by ppgrainbow** [comment]
Most of the supported graphics cards (such as the (S3 Trio, Tseng ET4000 and Trident 8900D) have support for built in SVGA graphics card drivers.

Look here on how to use higher resolution drivers under OS/2 2.1x. One example is for the S3 Trio which supports up to 4 MB VRAM: http://www.oldcomputers.it/parts/cubix/ers/docs/sitocubix/os2.htm

What graphics card are you using under OS/2 2.1x?

**Reply by MAAR2003** [reply]
I'm very sorry for the late reply, but I haven't tested it yet so I will choose one of these graphic cards you mentioned above!

**Comment by rakslice** [comment, marked as answer]
I'm using the "[PCI] Tseng Labs  ET4000/w32p Rev. D (Diamond Stealth32)" with the driver from https://www.os2site.com/sw/drivers/video/tseng/tsengw32.zip and this works for me.

The steps to install are in the readme file on the disk image.  Once you've done the setup and the Display Driver Install, change the resolution in OS/2 in System Setup -> System -> Screen tab, but first go to the second page of the Screen settings and change the monitor to something that supports higher refresh rates (e.g. Multisync XE21) to get more resolution options on the first page. Shutdown OS/2 and restart for the new resolution to take effect.
